easybuild/easyconfigs/m/medaka/medaka-1.5.0-foss-2021a.eb

```diff
@@ -0,0 +1,60 @@
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Danilo Gonzalez <danilo.gonzalez@hpcnow.com>
+# License::     GPL-v3.0
+# Updated to foss-2020b to use with artic tool
+# J. Sassmannshausen (GSTT/NHS UK)
+# Updated to 1.5.0
+# Jasper Grimm (UoY)
+
+easyblock = 'PythonBundle'
+
+name = 'medaka'
+version = '1.5.0'
+
+homepage = 'https://github.com/nanoporetech/medaka'
+description = "medaka is a tool to create a consensus sequence from nanopore sequencing data."
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+builddependencies = [('Autotools', '20210128')]
+
+_minimap_ver = '2.20'
+dependencies = [
+    ('Python', '3.9.5'),  # includes cffi
+    # TensorFlow ~=2.5.2 required by medaka 1.5.0, see requirements.txt
+    ('TensorFlow', '2.5.3'),
+    ('Pysam', '0.16.0.1'),
+    ('SAMtools', '1.13'),
+    ('minimap2', _minimap_ver),
+    ('HTSlib', '1.12'),  # for tabix, bgzip
+    ('Racon', '1.4.21'),
+    ('edlib', '1.3.9'),
```

Comment:
In `medaka-1.4.3-foss-2020b.eb` I also see `spoa` as a dependency. Any reason why it's not a dependency for this version of `medaka`? I'm guessing `pyspoa` is a python interface to `spoa`, so that would suggest that `spoa` is still required?

Reply:
`medaka` depends on `pyspoa`, the easyconfig for which already pulls in a suitable `spoa` dependency

Reply:
Ah, makes sense :)